echidna/output/plot_root.py

```diff
@@ -1,3 +1,4 @@
+from echidna.core import root_help
```

Comment:
Same as above, there are also no command line args here. Am I missing something in the way this works?